examples/cull-idle/README.md

```diff
@@ -0,0 +1,40 @@
+# `cull-idle` Example
+
+The `cull_idle_servers.py` file provides a script to cull and shut down idle
+single-user notebook servers. This script is used when `cull-idle` is run as
+a Service or when it is run manually.
+
+
+## Configure `cull-idle` to run as a Managed Service
+
+In `jupyterhub_config.py`, add the following dictionary for the `cull-idle`
+Service to the `c.JupyterHub.services` list:
+
+```python
+c.JupyterHub.services = [
+    {
+        'name': 'cull-idle',
+        'admin': True,
+        'command': 'python cull_idle_servers.py --timeout=3600'.split(),
+    }
+]
+```
+
+where `'admin': True` indicates this is a Managed Service and `command` is
```

Comment:
`admin: True` indicates that the service has admin permissions, it is not related to whether the service is managed or not. The command indicates that it is a service to be managed by the Hub.